rel/i18n/emqx_schema.hocon

```diff
@@ -124,6 +124,35 @@ mqtt_max_qos_allowed.desc:
 mqtt_max_qos_allowed.label:
 """Max QoS"""
 
+mqtt_subscription_max_qos_rules.desc:
+"""Maximum Subscription QoS allowance rules. Useful to limit QoS level requested by the client in a `SUBSCRIBE` packet. Take precedence over `max_qos_allowed` option, which then acts as the default when no rule applies."""
```

Comment:
It would be nice to have an example configuration.

Reply:
Added in 455059b201b4a03753ecd12aa5892aaa0f0c97a7. Please tell if examples need refinement.